packages/vscode-extension/src/debug/depsChecker/doctorConstant.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export const doctorConstant = {
+  Tick: "√",
+  Cross: "×",
+  WhiteSpace: "   ",
```

Comment:
Why three spaces?

Reply:
It's a proposal and maybe change to others after collecting feedback.